.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -80,6 +80,18 @@ as well as an upgrade of the previous version of Teleport.
     the installation to the current Teleport version (one major at a time) and try to
     log in using the U2F device as your second factor - it should work.
 
+    Windows Webauthn requires Windows 10 19H1 and device capable of Windows
+    Hello.
+
+  - [ ] Adding Users Webauthn Yubikey on Windows
+  - [ ] Adding Users Webauthn Windows Hello on Windows
+  - [ ] Login with MFA on Windows
+    - [ ] Add an OTP, a WebAuthn Yubikey and a Windows hello device with `tsh mfa add`
+    - [ ] Login via OTP
+    - [ ] Login via WebAuthn Yubikey
+    - [ ] Login via Windows Hello
+    - [ ] Login via WebAuthn using an U2F device
```

Comment:
> "Touch ID / Windows Hello" wherever it says "Touch ID" today?

I am not sure, it can happen that it will be tested by 2 different people since not everyone has windows machine, and there will be only 1 checkbox? 🤔

Reply:
Dup the sections for each OS, then?

Reply:
Or maybe we could have a "base" section and only dupe certain parts for each OS, so it's a bit more tidy?